Hymns/Display/DisplayHymnView.swift

```diff
@@ -38,73 +27,51 @@ struct DisplayHymnView: View {
     }
 }
 
-enum HymnLyricsTab: String {
-    case lyrics
-    case chords
-    case guitar
-    case piano
-}
-
-extension HymnLyricsTab {
-    var label: String {
-
-        switch self {
-        case .lyrics:
-            return "Lyrics"
-        case .chords:
-            return "Chords"
-        case .guitar:
-            return "Guitar"
-        case .piano:
-            return "Piano"
-        }
-    }
-}
-
 #if DEBUG
 struct DisplayHymnView_Previews: PreviewProvider {
-
     static var previews: some View {
 
         let loading = DisplayHymnView(viewModel: DisplayHymnViewModel(hymnToDisplay: PreviewHymnIdentifiers.hymn1151))
 
         let classic40ViewModel = DisplayHymnViewModel(hymnToDisplay: PreviewHymnIdentifiers.hymn40)
-
         classic40ViewModel.title = "Hymn 40"
         classic40ViewModel.isFavorited = true
-        classic40ViewModel.lyrics = [Verse]()
-        classic40ViewModel.chordsUrl = URL(string: "http://www.hymnal.net/en/hymn/h/40/f=gtpdf")
-        classic40ViewModel.guitarUrl = URL(string: "http://www.hymnal.net/en/hymn/h/40/f=pdf")
-        classic40ViewModel.pianoUrl = URL(string: "http://www.hymnal.net/en/hymn/h/40/f=ppdf")
         let classic40Lyrics = HymnLyricsViewModel(hymnToDisplay: PreviewHymnIdentifiers.hymn40)
-        classic40Lyrics.lyrics = [VerseViewModel(verseNumber: "1", verseLines: classic40_preview.lyrics[0].verseContent),
-                                  VerseViewModel(verseLines: classic40_preview.lyrics[1].verseContent),
-                                  VerseViewModel(verseNumber: "2", verseLines: classic40_preview.lyrics[2].verseContent),
-                                  VerseViewModel(verseNumber: "3", verseLines: classic40_preview.lyrics[3].verseContent),
-                                  VerseViewModel(verseNumber: "4", verseLines: classic40_preview.lyrics[4].verseContent)]
```

Comment:
I think we need these? Otherwise all the previews will have just the empty loading screen

Reply:
I fixed this for you, but can you fix the unit tests?